Habeeb/task_manager.py

```diff
@@ -0,0 +1,85 @@
+import task
+import utils
+
+urgenList=[]
+normalList=[]
```

Comment:
Why do you think you need `urgentList` and `normalList`? 
You only use the `taskList` in your operations and don't append any elements into them.